### ISSUE TITLE
Make Tasks an own Docs Plugin

### ISSUE THREAD

**Issue by mambax** [nds-swe/swdt#1]
Convert the tasks into a https://docusaurus.io/docs/docs-multi-instance

**Comment by Rishabhraghwendra18** [comment]
Hi...I am new to open source contribution . May you tell me what tasks are ? 😅

**Comment by mambax** [comment]
Hi, may I ask why you are interested in learning this here and how you found me? 😂🎉

I am preparing a lecture for a school and tasks are the students homeworks... 🤓

**Comment by Rishabhraghwendra18** [comment]
😂 I found this issue through issue tab . You are teaching them Open Source contribution ?

**Comment by mambax** [comment]
No, Software Engineering. But how did you find my repo at all? 👍🏼😂

**Comment by Rishabhraghwendra18** [comment]
Ok...May  I also be your student ?😜 . I found your repo from above "issue" tab in navbar of GitHub

**Comment by mambax** [comment]
Of my slides and docs yes. Sadly I can not invite you to the lectures as they are part of a conplete course at a school.

Feel free to try the Tasks none the less 😀
